ilc/.mocharc.json

```diff
@@ -0,0 +1,6 @@
+{
+    "spec": "./spec/**/*.spec.js",
```

Comment:
What about `**/*.spec.js`? So we can place tests near code.

Reply:
Moved